components/shared/AppSidebar/AppSidebar.tsx

```diff
@@ -73,16 +75,15 @@ export const AppSideBar = () => {
               })),
             ]}
             position="popper"
-            value={workspaceId}
+            value={workspaceId ?? "new"}
             placeholder="Select a workspace"
             onValueChange={(value) => {
               if (value === "new") {
                 router.push("/workspaces/new");
                 return;
               }
 
-              setWorkspaceId(value);
-              window.location.href = `/workspaces/${value}/settings`;
+              window.location.href = `/workspaces/${value}/repositories`;
```

Comment:
Now when you select a workspace from the app sidebar, it loads the dashboard instead of the settings page.